src/commands.hh

```diff
@@ -6,7 +6,10 @@ namespace Kakoune
 
 void register_commands();
 
-struct kill_session {};
+struct kill_session
+{
```

Comment:
We dont need this status field anymore, do we ?

Reply:
Actually, don't we need to set `local_client_exit` so that the server itself can exit with the status code stored in the exception?

Reply:
local_client_exit is set by the client m_on_exit callback (see line 700).

Reply:
Fixed in the latest push.